### PR TITLE
Changes for UM40

### PR DESCRIPTION
**Pull request by nothingneko**

### CODE REVIEW

**Comment by nothingneko** [comment]
Don't merge until lea approval

**Comment by nothingneko** [comment]
Todo: Add Gparted screenshot, hide readymade page for now

**Review by korewaChino — APPROVED** [review]
l gtm

**Review by lleyton — APPROVED** [review]

**Pull request merged**
